requirements.txt

```diff
@@ -0,0 +1,3 @@
+# Tested versions of specific packages used in openeo[localprocessing]
+openeo-pg-parser-networkx==2023.3.1
+openeo-processes-dask==2023.3.2
```

Comment:
This looks, with a filename like `requirements.txt`, the official list of dependencies for all users.
Also, some tools could automatically pick this up wrongly.
I would at least call it `requirements-localprocessing.txt` and put a more clear comment at the top (e.g. also mentioning this is an exprimental, optional feature)

Reply:
Done! Thanks for the clarification, I was in fact unsure about the naming of the file.